docroot/themes/humsci/su_humsci_gin_admin/dist/su_humsci_gin_admin.css

```diff
@@ -266,3 +266,13 @@ form[data-has-imported-fields] .ck.ck-editor__main>.ck-read-only {
   padding-inline: 0.5rem;
   max-width: 100%;
 }
+
+/* Styles to fix in the "Add more components" button */
+.paragraphs-add-dialog .paragraphs-ee-buttons .paragraphs-button--add-more {
+  border-width: 1px !important;
+}
+
+[data-once*="ginAccent"] .paragraphs-add-dialog .paragraphs-ee-buttons .paragraphs-button--add-more:hover,
+[data-once*="ginAccent"] .paragraphs-add-dialog .paragraphs-ee-buttons .paragraphs-button--add-more:focus {
```

Comment:
Lets avoid important here given the gin theme isn't giving the existing style an important. Change the first selector to `.gin--edit-form[data-once*="ginAccent"]` should override the original specificity.